wazo_call_logd/cel_interpretor.py

```diff
@@ -36,6 +36,8 @@
     rf'/var/lib/wazo/sounds/tenants/({UUID_REGEX.pattern})/monitor/({UUID_REGEX.pattern}).wav'
 )
 
+DESTINATION_KEYS_REGEX = re.compile(r'(?:^|,)(\w+):')
+
```

Comment:
You could probably reuse `KEY_PAIR_REGEX` and `parse_key_pair_sequence` instead. See `_extract_originate_all_lines_variables` for example.